examples/s2_field_combination_extern_rf_train/nrw_crop_extern_s2_workflow_to_datasets_with_timeshift.ipynb

```diff
@@ -0,0 +1,2364 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multi step join of monthly Sentinel-2 data to points of interest"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Setup\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook rquires the following packages:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geopandas as gpd\n",
+    "import geoengine as ge"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, connect to the Geo Engine instance:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ge.initialize(\"http://localhost:3030/api\", credentials=(\"admin@localhost\", \"admin1234\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Server:              http://localhost:3030/api\n",
+       "User Id:             d5328854-6190-4af9-ad69-4e74b0961ac9\n",
+       "Session Id:          0c7cd3ef-40bb-439f-a4b3-13049d1578e6\n",
+       "Session valid until: 2023-04-08T19:25:16.699Z"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "session = ge.get_session()\n",
+    "user_id = session.user_id\n",
+    "session"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Download Sentinel-2 data and store it in the Geo Engine\n",
+    "\n",
+    "Since the Sentinel-2 data is stored in the cloud (AWS S3), we create a workflow to store it as a local dataset. While this is not necessary, it allows us to use the data in the Geo Engine without the need to download it again."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To track how much work is done, get the used quota:"
```

Comment:
Why would the user want to track this?

Reply:
i wanted to track it. I can remove it if you don't like it.